.circleci/config.yml

```diff
@@ -745,10 +784,35 @@ workflows:
           filters:
             branches:
               ignore:
+                - feature/etherspot
                 - feature/etherspot-transactions
```

Comment:
@IAmKio @poocart I was not sure about this branch in ignore, is this maybe a leftover and can be removed?